### DISCUSSION TITLE
Background style

### DISCUSSION THREAD

**Discussion by MatteoRobbiati** [qibogang/qibogang.github.io#48] [Ideas]
I am thinking about the background style, also following @AleCandido's suggestions. 

I have at least two different proposals:

1. to use a "notebook sheet" style, on which we can draw with SVG, an example follows:

![new-bg](https://user-images.githubusercontent.com/62071516/206443832-6fabb14d-713d-47cd-b461-113693270bee.png)

2. to use a loop scheme drawn thanks to [this tool](https://fffuel.co/) or something similar to this. An example follows:

![loop_theme](https://user-images.githubusercontent.com/62071516/206443777-f99494db-c954-4276-ab78-9a1e8a3508f9.png)

Another solution can be to use both of these styles (or others), adopting different graphical solutions according to the goal of the page.

In the end, I think that also Jadwiga must be added to this organization, because she is also working on the graphical part.

**Comment by scarrazza** [comment]
The notebook looks better to me, but obviously depends on which wallpaper you use in 2. Could you please post 1 using white background instead of yellow, or lighter yellow?

**Reply by MatteoRobbiati** [reply]
I agree! 
About 2, the amazing thing is that we can use pixels for drawing a theme connected with quantum and at the same time draw a professional pattern.

**Comment by alecandido** [comment]
In any case, also for the second, you should have a less contrasting background. Try to shift everything towards white, in order to improve the appearance of the foreground

**Comment by MatteoRobbiati** [comment]
The two following examples I think respect the suggestions of both of you @AleCandido and @scarrazza:

- Notebook:

![note](https://user-images.githubusercontent.com/62071516/206448861-4dbd4063-daae-423f-9887-afaa545d9db0.png)

- SVG B-G:

![Screenshot from 2022-12-08 13-39-54](https://user-images.githubusercontent.com/62071516/206449019-8d5d3ad4-0599-4a6f-9cd2-4841d2a32c93.png)

**Reply by scarrazza** [reply]
This SVG B-G is not bad at all, but for the front page I prefer the notebook style, do you have screenshots with both solutions for the front page?

**Reply by MatteoRobbiati** [reply]
SVG B-G fits really bad if the page presents some text. 

![Screenshot from 2022-12-08 13-50-26](https://user-images.githubusercontent.com/62071516/206451369-c0b365ed-4fc3-4cab-baed-46ffb970af71.png)

Maybe only if I put opacity really close to zero can be okay.
The notebook (this is a test version, I can change color and lines density) seems to be better.

![Screenshot from 2022-12-08 13-51-06](https://user-images.githubusercontent.com/62071516/206451622-cee71699-9d01-4518-8e83-68948e3a3509.png).

If we want, I can do a lot of experiments using the app I linked above.

In the end, a more classical notebook solution:

![Screenshot from 2022-12-08 13-59-59](https://user-images.githubusercontent.com/62071516/206453115-0191795d-a242-4a1b-8213-dc3b42df42a8.png)

**Reply by scarrazza** [reply]
Yeah, this last white notebook looks good, maybe with a finer spacing.
On the other hand, the SVG is not bad for the tutorial page.

**Reply by MatteoRobbiati** [reply]
Perfect, I agree with the choice. 
Once I will have the SVG version of the current Home Page images from David, I will convert all the backgrounds to this white sheet. 
The next step will be to write some customized sentence/equation/graph on the notebook.

Thank you for the feedback.

**Reply by alecandido** [reply]
Sorry to repeat myself, but make it even lighter. Background has to be the back, and for this you need a rather uniform color. Contrast should only be in the foreground.

**Reply by MatteoRobbiati** [reply]
> Sorry to repeat myself, but make it even lighter. Background has to be the back, and for this you need a rather uniform color. Contrast should only be in the foreground.

You mean the tutorial's one?

**Reply by alecandido** [reply]
All, in particular the "chosen" one (i.e. the notebook). Not to waste effort on many different attempts, we should focus on a single one, we will improve later on.
And a single one that should be the same for all pages.

In any case, check all "modern" websites (https://nextjs.org, https://www.apple.com, https://github.com/, https://www.microsoft.com/, or whatever looks modern to you). They all have uniform backgrounds. Usually white, or almost white (unless you use the dark mode).

**Reply by MatteoRobbiati** [reply]
The difference from our home-page and their home-pages is that I feel that we need to fill space. 
This is the reason why I purposed the notebook style (which personally I like, especially because it can be customized).

Moreover, I deeply was inspired by [pennylane.ai](https://pennylane.ai/) webpage.

Of course, I agree with make it lighter, but I would prefer to avoid a white/uniform background.

**Reply by alecandido** [reply]
It is a personal opinion, but here we are discussing "fashion". Fashion is shaped by the most popular designs, and my feeling is that to the average eye those backgrounds seems to just clutter the space.

Consider that there is never the need to fill the space: if we have less content, we should make a smaller home page, and distribute better what we have.
The content will also come with time, and we should grow continuously.
Also consider that, for part-time maintainers, less content is an advantage, not an issue: we have less material to organize, so we need to spend less time and make less choices.

I know what inspired you, but pay attention that PennyLane websites has been made by professional designers, and drawers as well.
Also notice that only the home-page has a background and where there is content, the background is always white, as suggested above.

**Reply by alecandido** [reply]
![image](https://user-images.githubusercontent.com/10830876/206469328-a065ec99-4120-4a56-abd3-ff1fec47d714.png)

**Reply by alecandido** [reply]
The only place where they use drawings is when the content clearly stands out (there they put a video, but you can consider any kind of uniform card).

![image](https://user-images.githubusercontent.com/10830876/206469637-2d628967-a106-4527-9854-99277532ceea.png)

**Comment by MatteoRobbiati** [comment]
Thank you very much for your comments Alessandro. I am understanding the topic.
